scripts/url-registration.ts

```diff
@@ -0,0 +1,185 @@
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Mojaloop Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Mojaloop Foundation
+ - Name Surname <name.surname@mojaloop.io>
+
+ * Google
+ - Steven Wijaya <stevenwjy@google.com>
+ --------------
+ ******/
+
+import { program } from 'commander'
+import axios from 'axios'
+import packageInfo from '../package.json'
+
+const { Enum } = require('@mojaloop/central-services-shared')
+const EndPoints = Enum.EndPoints
+
+interface RegistrationValue {
+  targetServiceId: string
+  type: string
+  endpointPath: string
+}
+
+const registrationValues: Array<RegistrationValue> = [
+  // PUT /participants
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_PARTICIPANT_PUT,
+    endpointPath: '/participants',
+  },
+  // PUT /parties/{Type}/{ID}
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_PARTIES_PUT,
+    endpointPath: '/parties/{{partyIdType}}/{{partyIdentifier}}',
+  },
+  // PUT /parties/{Type}/{ID}/error
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.FSPIOP_CALLBACK_URL_PARTIES_PUT_ERROR,
+    endpointPath: EndPoints.FspEndpointTemplates.PARTIES_PUT_ERROR,
+  },
+  // PUT /consentRequests/{ID}
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.TP_CB_URL_CONSENT_REQUEST_PUT,
+    endpointPath: EndPoints.FspEndpointTemplates.TP_CONSENT_REQUEST_PUT,
+  },
+  // PUT /consentRequests/{ID}/error
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.TP_CB_URL_CONSENT_REQUEST_PUT_ERROR,
+    endpointPath: EndPoints.FspEndpointTemplates.TP_CONSENT_REQUEST_PUT_ERROR,
+  },
+  // POST /consents
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.TP_CB_URL_CONSENT_POST,
+    endpointPath: EndPoints.FspEndpointTemplates.TP_CONSENT_POST,
+  },
+  // PUT /consents/{ID}
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.TP_CB_URL_CONSENT_PUT,
+    endpointPath: EndPoints.FspEndpointTemplates.TP_CONSENT_PUT,
+  },
+  // PUT /consents/{ID}/error
+  {
+    targetServiceId: 'central-ledger',
+    type: EndPoints.FspEndpointTypes.TP_CB_URL_CONSENT_PUT_ERROR,
+    endpointPath: EndPoints.FspEndpointTemplates.TP_CONSENT_PUT_ERROR,
+  },
+  // POST /authorizations
+  {
+    targetServiceId: 'central-ledger',
+    type: 'FSPIOP_CALLBACK_URL_TRX_REQ_SERVICE',
```

Comment:
not sure about this enum, does it exist?

Reply:
yes /authorizations resised temporary in transaction-request-service